frontends/open-discussions/src/lib/learning_resources.js

```diff
@@ -116,9 +117,9 @@ export const getStartDate = (
   if (object.platform === platforms.OCW) {
     return `${capitalize(objectRun.semester || "")} ${objectRun.year || ""}`
   } else if (objectRun.start_date) {
-    return moment(objectRun.start_date).format("MMMM DD, YYYY")
+    return moment(objectRun.start_date).format(DISPLAY_DATE_FORMAT)
```

Comment:
This is the main change in this PR... `DISPLAY_DATE_FORMAT = "MMMM D, YYYY"`. (D, not DD)